posts/2021/10/jupyter-community-calls.md

```diff
@@ -0,0 +1,100 @@
+<!--
+.. title: Jupyter community calls
+.. slug: jupyter-community-calls
+.. date: 2021-11-10
+.. author: Isabela Presedo-Floyd, Tony Fast
+.. tags: Jupyter, Community
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+# Jupyter community calls
+
+A framing for open source is that the software and code are kernels of community.
+The code, and its abstractions, unite developers and their patrons; 
+a struggle for growing/evolving open communities is to make sure these groups remain connected. 
+A lot of us showed up for the code, but hung around for the community. 
+We'll continue this work talking about the monthly Jupyter community calls, 
+and how they help all [jovyans](https://jupyter.readthedocs.io/en/latest/community/content-community.html#what-is-a-jovyan) stay connected.
+
+
+## Why community calls?
+
+[Project Jupyter has a rich community calendar](https://jupyter.org/events) spanning different projects in the ecosystem. 
+Albeit, these events are open to all, the content of the meeting can become daunting and exclusive. 
+
+This year, Quansight Labs took on the responsibility of reviving an existing series of events: 
+the Jupyter community calls. 
+The original monthly series, starting in [March 2019][first], gave the stage to those using Jupyter daily. 
+These events move everyone outside of Jupyter's enormous code ecosystem to enjoy the affordances that Jupyter brings to its patrons as a computational tool.
+
+Jupyter is a rapidly moving piece of technology, and staying up to date with the latest 🔥 takes significant labor. 
+The community calls ensure us a monthly check-in of the latest and greatest. 
+Sometimes these events digress into Jupyter's rich folk history all the while peering into future interactive computing technologies like real time collaboration[^rtc] or browser-based computing environments [^jupyter-lite].
+
+Having these events consistently encouraged folks to get their work in to share with others. 
+These calls also gave some structure through an otherwise chaotic pandemic.
+
+
+## The last dozen events
+
+We made sure to record these events for posterity, and you can check them out in the [community call YouTube playlist][playlist], as [written notes](https://jupyter.readthedocs.io/en/latest/community/community-call-notes/index.html), or by following any of the links below:
+
+| 2020 |     |
```

Comment:
they look really bad as markup, but when rendered it was the least worst view i could create.

Reply:
My comment was more about the HTML output this is using a table to put the data into layout rather than to put the data into semantically coherent groups. 

But to be honest, I don't yet know enough about how folks actually use screen readers, so I don't know if this particular table would present a challenge or upset their expectations. It just raises a red flag for me based purely on my current understanding, which is subject to much and imminent revision